app/models/user.rb

```diff
@@ -7,6 +7,21 @@ class User < ApplicationRecord
   has_many :following_users, foreign_key: :followee_id, class_name: 'Friend'
   has_many :followers, through: :following_users
 
-  validates_presence_of :name, :email, :phone_number
+  validates_presence_of :name, :email, :google_id
 
+  def accepted_friend_ids
```

Comment:
Do we want this to be dynamic so that we only need one method that does al three of these filters?

```
def friends_by_status(friend_status)
   followed_users.where(request_status: friend_status).pluck(:followee_id)
end
```